code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -1,399 +1,403 @@
-/mob/living/carbon/human/examine(mob/user)

```

Comment:
What the hell happened here? Were any actual changes made? If not, you may want to check this file out from master, or something.